include/util.h

```diff
@@ -1,3 +1,21 @@
+// Copyright (c) 2020 LunarG, Inc.
```

Comment:
I presume this file was identified as not otherwise having changed content this year?

Should it at least include "and PlutoVR Inc."?

Reply:
I'm never quite sure the appropriate level of coverage here, presumably our agreement of PlutoVR includes ownership of the Created Work.  Although I'm the only author of this file, since PlutoVR sponsored the effort, it probably should say "and PlutoVR Inc."

Reply:
Is `2020` the correct year here?

Reply:
yes.  The last non-licensing change is June 16, 2020.